msteams-platform/overview.md

```diff
@@ -137,7 +131,23 @@ You don't need to be an expert programmer to build a great Teams app. Try one of
    :::column-end:::
 :::row-end:::
 
-## Resources
+:::row:::
+   :::column span="2":::
+
+## Get ideas for your app
+
+Looking for inspiration? Browse our list of real-world scenarios and industry solutions to understand the ways Teams apps can help your users.
```

Comment:
Looking for app development inspiration? Browse our list of real-world scenarios and industry solutions with high fidelity concept mocks to understand various ways Teams apps can help your users.